src/utils/looper.ts

```diff
@@ -13,12 +14,15 @@ async function doProbes(config: Config) {
       const probRes = await probing(item)
       const validatedResp = validateResponse(item.alerts, probRes)
 
-      log('id:', item.id, '- status:', probRes.status, 'for:', item.request.url)
+      log(
+        `id: ${item.id} - status: ${probRes.status} for: ${item.request.url} -- ${probRes.config.extraData?.responseTime}`
+      )
 
-      await sendAlerts({
-        validations: validatedResp,
-        notifications: config.notifications,
-        url: item.request.url ?? '',
+      await getServerStatus({
```

Comment:
I believe this changes because we will send alert by using `calculateSuccessFailure` function called by `getServerStatus`. Can we create `getServerStatus` as dumb as get status only? So there is no side effect like sendAlerts?

Reply:
Done